internal/provider/harness_base_resource.go

```diff
@@ -217,7 +224,19 @@ func (r *BaseHarnessResource) do(ctx context.Context, req framework.CreateOrUpda
 		return diags
 	}
 
-	if skip, reason := r.skip(ctx, data.Inventory, data.Id.ValueString()); skip {
+	ctx, err := r.store.Logger(ctx, data.Inventory, "harness_id", data.Id.ValueString(), "harness_name", data.Name.ValueString())
+	if err != nil {
+		return []diag.Diagnostic{diag.NewErrorDiagnostic("failed to initialize logger(s)", err.Error())}
+	}
+
+	// At this point (update/create), we should have an inventory Set during ModifyPlan()
+	inv, ok := r.store.inv.Get(data.Inventory.Seed.ValueString())
+	if !ok {
+		return []diag.Diagnostic{diag.NewErrorDiagnostic("failed to get inventory", err.Error())}
+	}
+
+	if skip, reason := r.skip(ctx, inv, data.Id.ValueString()); skip {
+		req.Plan.SetAttribute(ctx, path.Root("skipped"), types.StringValue(reason))
```

Comment:
What does this do since the harness resource itself doesn't have a `skipped` field in its schema?

Reply:
touche! I guess nothing! this is an oversight I'll remove it